### PR TITLE
Remove VMcache client from IRAF

### PR DESCRIPTION
**Pull request by LHe496**
This PR removes the long-deprecated VMcache client from IRAF source code, avoiding the internet requests when launching IRAF.

Fixes: #415

### CODE REVIEW

File: .github/workflows/citest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 name: IRAF CI test
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```
Comment (olebole) [inline .github/workflows/citest.yml:4]: I don't think we need this?

**Comment by olebole** [comment]
One more comment: You didn't use a read e-mail address for the commits. This will make it hard to keep you linked to the change for proper contribution acknowledgement. This *may* be your intention, but if not you may better use a real e-mail here.

**Comment by LHe496** [comment]
Consider all this. Maybe open a new pr is a better choice. I'd like to close this one and open a new one.

**Comment by olebole** [comment]
You could also do a force-push.

**Pull request closed without merging**